data/net/minecraft/client/gui/screens/recipebook/RecipeBookPage.mapping

```diff
@@ -1,8 +1,16 @@
 CLASS net/minecraft/client/gui/screens/recipebook/RecipeBookPage
+	METHOD addListener (Lnet/minecraft/client/gui/screens/recipebook/RecipeBookComponent;)V
+		ARG 1 component
 	METHOD init (Lnet/minecraft/client/Minecraft;II)V
 		ARG 1 minecraft
 		ARG 2 x
 		ARG 3 y
+	METHOD listButtons (Ljava/util/function/Consumer;)V
+		ARG 1 consumer
+	METHOD mouseClicked (DDIIIII)Z
+		ARG 1 mouseX
```

Comment:
Order of arguments are: ``mouseX, mouseY, button, centerX, centerY, ?, ?``
Could @sciwhiz12 @alcatrazEscapee or @ChampionAsh5357 have a look at this as well?

Reply:
This checks out, as `long`s and `double`s take up two parameter slots (so the number after a `long`/`double` is unoccupied:
 - `0` is the implicit receiver, since this is a non-`static` method.
 - `1` and `2` are taken up by the first `double`: `mouseX`.
 - `3` and `4` are taken up by the second `double`: `mouseY`.
 - `5` is taken up by the first `int`: `button`.
 - ...and the rest are straightforward, since they're all `int`s.